src/legacy/ui/ui_render/ui_render_mixin.js

```diff
@@ -78,6 +78,8 @@ export function uiRenderMixin(kbnServer, server, config) {
     },
   });
 
+  server.exposeStaticDir('/bundles/core/{path*}', fromRoot('src/core/public/target/public'));
```

Comment:
Probably a better way to this. I wasn't sure where the routes for regular plugin bundles is located.